ensemble-forecast/qra-ensemble/quantile-ensembling.R

```diff
@@ -5,8 +5,6 @@
 library(magrittr)
 library(data.table)
 
-source(here::here("R", "get_us_data.R"))
-
 # load deaths data -------------------------------------------------------------
 death_data_inc <- load_observed_deaths(weekly = TRUE) %>%
   dplyr::rename(geography = region, 
```

Comment:
Weekly data needs to be changed over to epiweeks everywhere

Reply:
As above.